app/components/cards/card-latest-workflows.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { inject: {service}} = Ember;
+
+export default Ember.Component.extend({
+  session: service(),
```

Comment:
It my be my fault when I rebase and squashed the branch. Could you please fix it?